src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs

```diff
@@ -209,8 +215,12 @@ private void SetTransformMap(PropertyMap transformMap)
                 return;
             }
 
-            transformMap.Find((int)VisualTransformPropertyType.Offset)?.Get(Offset);
-            transformMap.Find((int)VisualTransformPropertyType.ExtraSize)?.Get(Extents);
+            PropertyValue temp = transformMap.Find((int)VisualTransformPropertyType.Offset);
+            temp?.Get(Offset);
+            temp.Dispose();
```

Comment:
PropertyMap.Find result can be null. Please add "?" or move them into if statement.

Reply:
> PropertyMap.Find result can be null. Please add "?" or move them into if statement.

Fixed